src/Controller/ReceiptController.php

```diff
@@ -118,16 +122,19 @@ public function create(Request $request)
             return $this->redirectToRoute('receipt_create');
         }
         // Else: User is viewing receipt page (no receipt exist: path=null)
-        else {
-            echo "<script>alert('The file is too large. Allowed maximum size is 2 MiB.)";
-            $receipt->setPicturePath("");
+        else{ $receipt->setPicturePath("");}
+
+        if ($form->isSubmitted() && !$form->isValid()&& $receipt->getPicturePath()=="") {
+
+            $this->addFlash('warning', "The file is too large. Allowed maximum size is 2 MiB.)");
```

Comment:
Rememeber that this page is in norwegian. Error messages shown to user should be as well.